controller/src/main.py

```diff
@@ -34,25 +41,59 @@ def cli():
 # click command for initializing the database tables
 @click.command()
 def initdb():
-    """Initialize the database (delete old tables)"""
+    """
+    Initialize the database tables (delete old tables).
```

Comment:
why all the default data source, default workflow?

Reply:
it's not a must of course, only convenient for our dev environment